_layouts/post.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 ---
-
+<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico?">
```

Comment:
Am aware it doesn't stop file from resolving but it's misleading. Also make sure you keep your branch up to date with master!